PerformanceMetrics/CMakeLists.txt

```diff
@@ -28,12 +28,12 @@ set(MODULE_NAME ${NAMESPACE}${PROJECT_NAME})
 message("Setup ${MODULE_NAME} v${PROJECT_VERSION}")
 
 set(PLUGIN_PERFORMANCEMETRICS_AUTOSTART "true" CACHE STRING "Automatically start Performance Metrics plugin")
-set(PLUGIN_PERFORMANCEMETRICS_LOGGER_IMPLEMENTATION "TRACING" CACHE STRING "Defines what implementation to use for logging the Performance Metrics")
+set(PLUGIN_PERFORMANCEMETRICS_LOGGER_IMPLEMENTATION "SYSLOG" CACHE STRING "Defines what implementation to use for logging the Performance Metrics")
 
 # Plugins built from this repository that can be autmatically enabled or enabled manually when built externally
 set(PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER "${PLUGIN_WEBKITBROWSER}" CACHE BOOL "Enable monitor for the Performance Metrics plugin")
 set(PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER_APPS "${PLUGIN_WEBKITBROWSER_APPS}" CACHE BOOL "Enable monitor for the Performance Metrics Apps plugin")
-set(PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER_RESIDENT_APP "${PLUGIN_WEBKITBROWSER_RESIDENT_APP}" CACHE BOOL "Enable monitor for the Performance Metrics Resident App plugin")
+#set(PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER_RESIDENT_APP "${PLUGIN_WEBKITBROWSER_RESIDENT_APP}" CACHE BOOL "Enable monitor for the Performance Metrics Resident App plugin")
```

Comment:
PLUGIN_WEBKITBROWSER_RESIDENT_APP is global flag which is used in webkitbrowser plugin as well.
We will not be able to set this flag to OFF. This will disable the ResidentApp plugin. 
Need to look for other options for this.